modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_kazimi/tests

```diff
@@ -0,0 +1,63 @@
+[Tests]
+  design = 'ADWallHeatTransferCoefficientKazimiMaterial.md'
+  issues = '#24340'
+  [normal_operation]
+    requirement = "The system shall compute the heat conductance using the Kazime-Carelli "
+                  "correlation for the Nusselt number"
+    type = 'CSVDiff'
+    input = 'kazimi_test.i'
+    csvdiff = 'kazimi_test_out.csv'
+  []
+  [abnormal_operation]
+    requirement = "The system shall return a warning message when: "
+    [out_of_range_warning_1]
+      type = 'RunApp'
+      input = 'kazimi_test.i'
+      prereq = 'test/normal_operation'
+      cli_args = "PoD=1.05"
+      expect_out = "The Kazimi-Carelli correlation is valid when Pe is between 10 and 5000, and P/D "
+                   "is between 1.1 and 1.4. Be aware that using values out of this range may lead to "
+                   "significant errors in your results!"
+      method = 'OPT'
```

Comment:
Remove this line (here and in the other spec)

Reply:
done